test/test_managercli.py

```diff
@@ -32,108 +31,6 @@ def test_entitlement_for_not_installed_product_shows_nothing(self):
         # no product certs installed...
         self.assertEqual(0, len(product_status))
 
-    def test_entitlement_for_installed_product_shows_subscribed(self):
-        product_directory = StubProductDirectory(pids=["product1"])
-        provide(PROD_DIR, product_directory)
-        ent_cert = StubEntitlementCertificate("product1")
-
-        stub_sorter = StubCertSorter()
-        stub_sorter.valid_products["product1"] = [ent_cert]
-        provide(CERT_SORTER, stub_sorter)
-
-        product_status = InstalledProducts(StubUEP()).list()
-
-        self.assertEqual(1, len(product_status))
-        self.assertEqual("subscribed", product_status[0][4])
```

Comment:
I would keep at least one unit test for installed product, when system is registered, because we still support listing of installed products. It will be necessary to modify this unit tests. Of course, it is OK to delete other cases, because we do not support these cases in pure SCA world.